src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,28 @@
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import React from 'react';
+import { Comment } from '../../types/Comment';
+
+import './CommentInfo.scss';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <li key={comment.id} className="comments__text">
```

Comment:
```suggestion
    <li className="comments__text">
```

redundant, you can use key directy on component call (look at my comment  in the ComponentList)